packages/app/src/cli/models/extensions/schemas.ts

```diff
@@ -32,18 +32,17 @@ export const ApiVersionSchema = zod.string()
 
 export type ApiVersionSchemaType = zod.infer<typeof ApiVersionSchema>
 
+export const FieldSchema = zod.object({
+  key: zod.string().optional(),
+  name: zod.string().optional(),
+  description: zod.string().optional(),
+  required: zod.boolean().optional(),
+  type: zod.string(),
+  choices: zod.array(zod.string()).optional(),
```

Comment:
Probably don't want `choices` here in the FieldSchema since it's not supported by all extensions.